azure-communication-ui/azure-communication-ui-demo-app/src/main/java/com/azure/android/communication/ui/callingcompositedemoapp/features/SettingsFeatures.kt

```diff
@@ -5,78 +5,64 @@ package com.azure.android.communication.ui.callingcompositedemoapp.features
 
 import android.content.Context
 import android.util.LayoutDirection
+import com.azure.android.communication.ui.callingcompositedemoapp.BaseApplication
 import com.azure.android.communication.ui.callingcompositedemoapp.DEFAULT_LANGUAGE_VALUE
+import com.azure.android.communication.ui.callingcompositedemoapp.DEFAULT_LOCALE_CODE
 import com.azure.android.communication.ui.callingcompositedemoapp.DEFAULT_RTL_VALUE
 import com.azure.android.communication.ui.callingcompositedemoapp.LANGUAGE_ADAPTER_VALUE_SHARED_PREF_KEY
-import com.azure.android.communication.ui.callingcompositedemoapp.LANGUAGE_CUSTOM_TRANSLATION_ENABLE
 import com.azure.android.communication.ui.callingcompositedemoapp.LANGUAGE_ISRTL_VALUE_SHARED_PREF_KEY
+import com.azure.android.communication.ui.configuration.LanguageCode
+import com.azure.android.communication.ui.configuration.LocalizationConfiguration
 import com.azure.android.communication.ui.utilities.implementation.FEATURE_FLAG_SHARED_PREFS_KEY
+import java.util.Locale
 
 class SettingsFeatures {
 
     companion object {
-        var isLanguageFeatureEnabled: Boolean = false
 
-        // Language Features
-        fun language(context: Context): String? {
-            return context.applicationContext
+        fun language(): String? {
+            return BaseApplication.getInstance()
```

Comment:
You need BaseApplication only so that you can carry out SharedPreference operations, correct?
If so, will following the example in FeatureFlags.kt to obtain a reference to `SharedPreference` work in this case?

Reply:
Thanks Albert :)